relay-dynamic-config/Cargo.toml

```diff
@@ -15,6 +15,7 @@ default = []
 [dependencies]
 anyhow = { workspace = true }
 assert-json-diff = "2.0.2"
+itertools = { workspace = true }
```

Comment:
It looks like this dependency is not actually used, can we remove it?

Reply:
we use it on `.unique()`, but i'll try to remove that function anyway like you mentioned, in which case i'll remove the dependency